src/lib_ccx/matroska.h

```diff
@@ -172,7 +172,7 @@ char* matroska_track_text_subtitle_id_strings[] = {
 char* matroska_track_text_subtitle_id_extensions[] = {
         "srt", "ssa", "ass",
         "usf", "vtt", "bmp",
-        NULL, NULL  // Unknown
+        "idx", NULL  // Unknown
 };
 
```

Comment:
Are yo sure replacing that NULL doesn't break things? I don't know why we had two, maybe it's some marker...(I haven't looked into the code).

Reply:
> Are yo sure replacing that NULL doesn't break things?

It's just used once, to write the filename with its extension.

>I don't know why we had two, maybe it's some marker...(I haven't looked into the code).

It is a mapping of `codec_id` to their file extensions. Last 2 codecs are `VOBSUB` and `KATE`. I guess their extensions were previously unknown, hence kept as null.